scss/_base.scss

```
@@ -0,0 +1,33 @@
+/*------------------------------------------------
+    ファイル名：_base.scss
+    機能：ベーススタイルを記述する
+--------------------------------------------------*/
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+
+    color: #222222;
+    font-family: "Noto Serif", "Noto Serif JP", sans-serif;
```

セリフ体を指定しているので、どちらもない場合のフォント指定もセリフ体にした方がいいのでは？と思いました。
```suggestion
    font-family: "Noto Serif", "Noto Serif JP", serif;
```
このように